assets/js/components/charts/Forecasts.jsx

```diff
@@ -103,7 +103,10 @@ export default class Forecasts extends Component<Props> {
       initialTime = new Date()
       lastTime = d3.timeMonth.offset(initialTime, 1)
     } else {
-      initialTime = d3.min(flatten, (d) => d.time)
+      initialTime = d3.timeDay.offset(
+        d3.min(flatten, (d) => d.time),
+        -1
+      )
```

Comment:
Also had to add this in order to get a plot like this:

![image](https://user-images.githubusercontent.com/13782680/223182436-aade2b5c-8d5f-48aa-bde7-bdc1d6777de3.png)

Instead of an out-of-sight first point as in the original comment's GIF (actually, this wasn't correct before, hiding some data).